packages/ui-informed/src/index.ts

```diff
@@ -1,4 +1,31 @@
+/**
+ * FormWithControls
+ */
+export * as FormWithControls from './form';
```

Comment:
I am on the edge with this one, I am leaning towards not defining these aliases in this case and simply having the application use `import * as Form from '@utima/ui-informed';`.

There would be cases like `Form.Form` but you could also define custom alias, or use it without one. WHat do you think?

Reply:
I just don't like that if you let it only on the import in final application, then there will be thinks like `Form.useDefaultSubmitActions`. But I admit that it's maybe kinda unusual to create this named import inside of package, but on the other hand we are directly offering user of our package "the best way" how to use it and do not mess up importing Checkbox from UI insted of UI-informed :-D 

But I don't have strong opinion on that, I just stated my thoughts and leaving decision up to you (so far I'm not removing it)